lua/nvim-treesitter/parsers.lua

```diff
@@ -284,11 +284,25 @@ function M.get_parser_configs()
   return M.list
 end
 
+local parser_files
+
+function M.reset_cache()
+  parser_files = setmetatable({}, {
+    __index = function(tbl, key)
+      rawset(tbl, key, api.nvim_get_runtime_file('parser/' .. key .. '.*', false))
+      return rawget(tbl, key)
+    end
+  })
+end
+
+M.reset_cache()
```

Comment:
should we call this `init_parser_cache()` ? i'm not too sure, maybe it doesn't really matter.